src/applications/check-in/pre-check-in/tests/e2e/posting-answers/answered.yes.to.both.questions.cypress.spec.js

```diff
@@ -0,0 +1,70 @@
+import { generateFeatureToggles } from '../../../api/local-mock-api/mocks/feature.toggles';
+import '../support/commands';
+import Timeouts from 'platform/testing/e2e/timeouts';
+
+import validateVeteran from '../pages/ValidateVeteran';
+import introduction from '../pages/Introduction';
+import NextOfKin from '../pages/NextOfKin';
+import Demographics from '../pages/Demographics';
+import apiInitializer from '../support/ApiInitializer';
+
+describe('Pre-Check In Experience ', () => {
+  let apiData = {};
+  beforeEach(function() {
+    cy.intercept('GET', '/v0/feature_toggles*', generateFeatureToggles({}));
+    apiInitializer.initializeSessionGet.withSuccessfulNewSession();
+
+    apiInitializer.initializeSessionPost.withSuccess();
+
+    apiData = apiInitializer.initializePreCheckInDataGet.withSuccess();
+
+    apiInitializer.initializePreCheckInDataPost.withSuccess(req => {
+      expect(req.body.preCheckIn.uuid).to.equal(
+        '0429dda5-4165-46be-9ed1-1e652a8dfd83',
+      );
+      expect(req.body.preCheckIn.demographicsUpToDate).to.equal(true);
+      expect(req.body.preCheckIn.nextOfKinUpToDate).to.equal(true);
+    });
+  });
+  afterEach(() => {
+    cy.window().then(window => {
+      window.sessionStorage.clear();
+    });
+  });
+  it('Answered yes to both questions', () => {
+    cy.visitPreCheckInWithUUID();
+    // page: Validate
+    validateVeteran.validatePageLoaded();
+    validateVeteran.validateVeteran();
+    cy.injectAxe();
+    cy.axeCheck();
+
+    validateVeteran.attemptToGoToNextPage();
+
+    // page: Introduction
+    introduction.validatePageLoaded();
+    introduction.countAppointmentList(apiData.payload.appointments.length);
+    cy.injectAxe();
+    cy.axeCheck();
+    introduction.attemptToGoToNextPage();
+
+    // page: Demographics
+    Demographics.validatePageLoaded();
+    cy.injectAxe();
+    cy.axeCheck();
+    Demographics.attemptToGoToNextPage();
+
+    // page: Next of Kin
+    NextOfKin.validatePageLoaded();
+    cy.injectAxe();
+    cy.axeCheck();
+    NextOfKin.attemptToGoToNextPage();
+
+    // page: Confirmation
+    cy.get('h1', { timeout: Timeouts.slow })
```

Comment:
Should this one use the Confirmation page helper class here? That feature just got merged a little while ago.

Reply:
Good Catch!